sdk/skills/create-mcp-eval/references/project-setup.md

```
@@ -0,0 +1,79 @@
+# For OAuth-protected servers:
+# MCP_REFRESH_TOKEN=...
+# MCP_CLIENT_ID=...
+# MCP_CLIENT_SECRET=...
+
+# Save eval results to MCPJam (optional)
+# MCPJAM_API_KEY=sk_...
+# MCPJAM_PROJECT_ID=<project id>  # optional; defaults to your org’s Default project
+```
+
+### .gitignore additions
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)